README.md

```diff
@@ -80,6 +82,33 @@ And export them to vim plugin format via:
 
 All merges should be done on dev branch before being merged onto master
 
+# Additional helpful tips for Vim users
+
+You need not use `Vundle`. Just go to `~/.vim/bundle` and clone `ensime-vim`.
+
```

Comment:
Just a thought for people who already have Pathogen. Could be a note for people who already have Pathogen.